pallets/ds-accounts/src/lib.rs

```diff
@@ -34,22 +37,30 @@ pub mod prelude {
 /// Structure, specific for each role
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(Encode, Decode, Clone, Default)]
-pub struct Account<Moment, AccountRole> {
+pub struct Account<Moment, AccountRole, AccountManager> {
     pub roles: AccountRole,
     pub create_time: Moment,
```

Comment:
"Moment" is probably not a best word here. The word "instant" is more commonly used.

Reply:
Moment  is widely used  in  substrate pallets term as a sdefault  template name for Time values

Reply:
Should I return "Moment" or can I leave "Instant"?

Reply:
> Moment is widely used in substrate pallets term
A bit strange word to me, but ok..

Reply:
> Should I return "Moment" or can I leave "Instant"?

Better to use the same word everywhere, whether it Instant or Moment.